src/test/java/io/stargate/sgv2/jsonapi/exception/ExceptionHandlerTest.java

```diff
@@ -22,13 +21,13 @@ public class ExceptionHandlerTest {
   public void handleNull() {
 
     var handler =
-        new ExceptionHandler<TableSchemaObject, UnsupportedOperationException>() {
+        new ExceptionHandler<UnsupportedOperationException>() {
           @Override
           public Class<UnsupportedOperationException> getExceptionClass() {
             return UnsupportedOperationException.class;
           }
         };
-    var actualEx = assertDoesNotThrow(() -> handler.maybeHandle(null, null));
```

Comment:
I've always wondered the point of "assertDoesNotThrow" -- why not just execute; if exception is thrown, test will still fail? Or does `assertDoesNotThrow()` provide some useful extra information?

Reply:
it's pretty useless, and the docs say as much - any thrown error causes a test to fail. I tend to find my self using it when writing tests when I want to get the the result of the function and I am trying to be very declarative about what is expected.

Reply:
Right, but in theory you could say that every line that does not expect exception should be surrounded by "assertDoesNotThrow".

It's fine, I just wanted to ask as I've seen it be used -- could have missed something.